nitro-protocol/contracts/ForceMove.sol

```diff
@@ -116,42 +116,37 @@ contract ForceMove is IForceMove, StatusManager {
+     * @notice Finalizes a channel according to the given finalState. External wrapper for _conclude.
+     * @dev Finalizes a channel according to the given finalState. External wrapper for _conclude.
      * @param fixedPart Data describing properties of the state channel that do not change with state updates.
-     * @param proof An ordered array of structs, that can be signed by any number of participants, each struct describing the properties of the state channel that may change with each state update. The proof is a validation for the supplied candidate.
-     * @param candidate A struct, that can be signed by any number of participants, describing the properties of the state channel to change to. The candidate state is supported by proof states.
+     * @param finalState A state, signed by all participants, describing the closing state of the channel.
      */
-    function conclude(
-        FixedPart memory fixedPart,
-        SignedVariablePart[] memory proof,
-        SignedVariablePart memory candidate
-    ) external virtual override {
-        _conclude(fixedPart, proof, candidate);
+    function conclude(FixedPart memory fixedPart, SignedVariablePart memory finalState)
+        external
+        virtual
+    {
+        _conclude(fixedPart, finalState);
     }
 
     /**
-     * @notice Finalizes a channel by providing a finalization proof. Internal method.
-     * @dev Finalizes a channel by providing a finalization proof. Internal method.
+     * @notice Finalizes a channel according to the given finalState. Internal method.
+     * @dev Finalizes a channel according to the given finalState. Internal method.
      * @param fixedPart Data describing properties of the state channel that do not change with state updates.
-     * @param proof An ordered array of structs, that can be signed by any number of participants, each struct describing the properties of the state channel that may change with each state update. The proof is a validation for the supplied candidate.
-     * @param candidate A struct, that can be signed by any number of participants, describing the properties of the state channel to change to. The candidate state is supported by proof states.
+     * @param finalState A state, signed by all participants, describing the closing state of the channel.
      */
-    function _conclude(
-        FixedPart memory fixedPart,
-        SignedVariablePart[] memory proof,
-        SignedVariablePart memory candidate
-    ) internal returns (bytes32 channelId) {
+    function _conclude(FixedPart memory fixedPart, SignedVariablePart memory finalState)
+        internal
+        returns (bytes32 channelId)
+    {
         _requireCorrectChainId(fixedPart.chainId);
         channelId = NitroUtils.getChannelId(fixedPart);
 
         // checks
         _requireChannelNotFinalized(channelId);
-        require(proof.length == 0, 'Must submit exactly 1 state');
```

Comment:
This is the motivating line. If the passed in proof must be empty, why pass it in?